indico/modules/events/templates/footer.html

```diff
@@ -16,26 +16,8 @@ <h4>{% trans %}Direct link{% endtrans %}</h4>
             </a>
             <div>
                 <h4>{% trans %}Social networks{% endtrans %}</h4>
-                <div>
-                    <span id="fb-loading">{% trans %}Loading...{% endtrans %}</span>
-                    <div id="fb-like" class="fb-like"
-                         data-href="{{ url }}"
-                         data-size="small"
-                         data-width="100"
-                         data-layout="standard"
-                         data-colorscheme="{{ 'dark' if dark else 'light' }}"
-                         data-action="like"
-                         data-show-faces="false"
-                         data-share="true"></div>
-                </div>
-                <div class="social-site">
-                    <div class="g-plusone"
-                         data-annotation="inline"
-                         data-width="220"
-                         data-href="{{ url }}"></div>
-                </div>
                 <div class="social-site">
-                    <a href="https://twitter.com/share" class="twitter-share-button" data-count="horizontal">
+                    <a href="https://twitter.com/intent/tweet?text={{ url }}" class="twitter-share-button">
```

Comment:
Would be nice to keep the event title in the text as well (like we have right now)